x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/factories/wrap_hits.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { estypes } from '@elastic/elasticsearch';
+import type { SecuritySharedParams, SignalSource, SimpleHit } from '../types';
+import { generateId } from '../utils/utils';
+import { transformHitToAlert } from './utils/transform_hit_to_alert';
+import type { BuildReasonMessage } from '../utils/reason_formatters';
+import type {
+  BaseFieldsLatest,
+  WrappedFieldsLatest,
+} from '../../../../../common/api/detection_engine/model/alerts';
+
+export const wrapHits = (
```

Comment:
This might be a good time to add a lil' JSDoc to explain what the heck this function's responsibilities are.